test/conftest.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from unittest.mock import MagicMock
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--api", action="store_true", help="runs the tests marked @pytest.mark.api"
+    )
```

Comment:
For politeness reasons?

Reply:
That, and also the api tests take considerably longer than all the other tests combined.